python/sbp/client/loggers/base_logger.py

```diff
@@ -25,7 +25,7 @@ class BaseLogger(object):
   filename : string
     File to log to.
   """
-  def __init__(self, filename, mode="w", tags={}, dispatcher=dispatch):
+  def __init__(self, filename, mode="w", tags={}, dispatcher=None):
     self.handle = open(filename, mode)
     self.dispatcher = dispatcher
```

Comment:
Yeah, what's the motivation for moving the message dispatch default to `None`?

It's just occurred to me that message dispatch could actually just another thing we compose in the current calling chain instead of passing as an argument. Something like: `Handler(Dispatcher(Framer(driver.read, driver.write)))`. @mfine, thoughts?